src/libraries/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs

```diff
@@ -36,6 +36,34 @@ public async Task CertificateValidationRemoteServer_EndToEnd_Ok()
             }
         }
 
+        // MacOS has has special validation rules for apple.com and icloud.com
+        [ConditionalTheory]
+        [OuterLoop("Uses external servers")]
+        [InlineData("www.apple.com")]
+        [InlineData("www.icloud.com")]
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public async Task CertificateValidationApple_EndToEnd_Ok(string host)
+        {
+            using (var client = new TcpClient())
+            {
+                try
+                {
+                    await client.ConnectAsync(host, 443);
+                }
+                catch (Exception ex)
+                {
+                    // if we cannot connect skip the test instead of failing.
+                    new SkipTestException($"Unable to connect to '{host}': {ex.Message}");
```

Comment:
@wfurt Needs `throw`.

Reply:
yes, you are right. I'll fix it. thanks.